eta/core/image.py

```diff
@@ -786,6 +786,17 @@ def from_dict(cls, d):
         return cls(images=images, schema=schema)
 
 
+class _BigImageLabelsSet(BigSet, _ImageLabelsSet):
+    '''Internal set type used by BigImageSetLabels to store labels.'''
+
+
+class BigImageSetLabels(ImageSetLabels):
+    '''ImageSetLabels which uses a BigSet (_BigImageLabelsSet) to store labels.
```

Comment:
No need to mention `_BigImageLabelsSet` explicitly. Users don't need to read about private classes.

I'd explicitly mention that `BigImageSetLabels` behaves identically to `ImageSetLabels` except that each element is stored on disk.